src/stylesheets/components/_header.scss

```diff
@@ -135,25 +55,17 @@
       }
     }
 
+    &:active:not(:hover),
+    &:focus:not(:hover) {
+      &::after {
+        border-color: $govuk-focus-text-colour;
```

Comment:
If you use `currentColor` I think you can simplify this to avoid so many selectors.

Reply:
If you press 'space' on the button you'll see this isnt quite right, whereas currentColor avoids having to figure this out.

Reply:
@aliuk2012 ^